DXMainClient/DXGUI/Multiplayer/GameLobby/MultiplayerGameLobby.cs

```diff
@@ -594,7 +607,11 @@ protected void PrintDiceRollResult(string senderName, int dieSides, int[] result
                 senderName, results.Length, dieSides, string.Join(", ", results)
             ));
         }
-
+        protected void PrintHandRPSResult(string senderName, string result)
+        {
+            AddNotice(String.Format("{0} {1}".L10N("Client:Main:PrintHandRPSResult"),
```

Comment:
This would not correctly localize the string since the parameters of `string.Format` are English strings